### ISSUE TITLE
"Generate Support Bundle" Button for Rancher

### ISSUE THREAD

**Issue by oxr463** [rancher/rancher#30918]
> You can click Generate Support Bundle link at the bottom of the UI to download a zip file contains Longhorn related configuration and logs.

Source: https://longhorn.io/docs/0.8.1/advanced-resources/troubleshooting

It would be great to have this for Rancher.

**Comment by rajivml** [comment]
Yeah it would be great to have this feature enabled for Rancher as well and there should be an config exposed like this on what all logs to collect as part of generating support bundle, so it will be extremely useful for OEM vendors who are packaging their software through Rancher 

https://help.replicated.com/api/support-bundle-yaml-specs/kubernetes-logs/

**Comment by wichitalineman** [comment]
Checking in on this request to add automated bundling for support logs on Rancher. Very manual process at the moment, which can lead to missed logs, which can lead to lost time trying to solve an issue. The functionality already exists for Longhorn, please port this over to Rancher. This request has been open for a year without any feedback at all. I believe it would be of great value for all Rancher users.
